java/pom.xml

```diff
@@ -75,6 +75,7 @@
         <maven-shade-plugin.version>3.2.4</maven-shade-plugin.version>
         <powsybl-dependencies.version>1.1.0</powsybl-dependencies.version>
         <powsybl-math-native.version>1.2.1</powsybl-math-native.version>
+        <powsybl-entsoe.version>1.4.0</powsybl-entsoe.version>
```

Comment:
You don't need to specify the version, it is imported from powsybl dependencies with the right one